### PR TITLE
Add yougetsignal in dns_search.py sources

### PR DESCRIPTION
**Pull request by adviksinghania**
For issue #86 
Added yougetsignal in modules/osint/dns_search.py

### CODE REVIEW

File: modules/osint/dns_search.py [hunk 1]
```diff
@@ -190,6 +190,25 @@ def dnsdumpster(self, q):
...
+	headers = {
+		'authority': 'domains.yougetsignal.com',
+		'accept': 'text/javascript, text/html, application/xml, text/xml, */*',
+		'origin': 'https://www.yougetsignal.com',
+		'referer': 'https://www.yougetsignal.com/',
+	}
+	data = {'remoteAddress': q}
+	try:
+		req = self.request('https://domains.yougetsignal.com/domains.php', method='POST', headers=headers, data=data).json()
+	except Exception:
```
Comment (z3y50n) [inline modules/osint/dns_search.py]: Make it `Exception as e` for consistency

File: modules/osint/dns_search.py [hunk 2]
```diff
@@ -190,6 +190,25 @@ def dnsdumpster(self, q):
...
+def yougetsignal(self, q):
+	self.verbose('[YOUGETSIGNAL] Searching in yougetsignal...')
+	headers = {
+		'authority': 'domains.yougetsignal.com',
+		'accept': 'text/javascript, text/html, application/xml, text/xml, */*',
+		'origin': 'https://www.yougetsignal.com',
+		'referer': 'https://www.yougetsignal.com/',
+	}
+	data = {'remoteAddress': q}
+	try:
```
Comment (z3y50n) [inline modules/osint/dns_search.py:202]: Use `try-except-else` statements. Look at other functions of the same module

**Comment by adviksinghania** [comment]
@z3y50n I've made the necessary changes to the script.
Please review it again, Thanks.

**Comment by z3y50n** [comment]
Replaced `map` with list comprehension and used `set_data` function to store the results. I am merging good job!

**Pull request merged**